packages/indexer/indexer/data.py

```diff
@@ -0,0 +1,42 @@
+import aiohttp
+from indexer.chain_mapper import CosmosAPI
+
+
+async def get_block(
+    session: aiohttp.ClientSession,
+    api: CosmosAPI,
+    height: str = "latest",
+    max_retries=5,
+) -> dict | None:
+    retries = 0
+    while retries < max_retries:
+        try:
+            async with session.get(
+                f"{api.url}/cosmos/base/tendermint/v1beta1/blocks/{height}"
```

Comment:
To keep things clean, maybe at the top of the script we have something like: `block_endpoint = f`{api.url}/cosmos/base/tendermint/v1beta1/blocks/`. In our current codebase, there has been several times where I've forgotten to update endpoints when they were mixed throughout the codebase. Keeping it at the top would be organized. 

Or if you are using objects, then we might as well just have that defined on the `CosmosAPI` object.

Reply:
I like this thought

Reply:
Another reason, is not all chains will be on `v1beta`. This depends on what version of tendermint they are running.